mods/ts/sequences/misc.yaml

```diff
@@ -1,12 +1,11 @@
 overlay:
 	Defaults:
-		Start: 1
+		Start: 0
 		Offset: 0, -12
 	build-valid-snow: place
-		Start: 0
 	build-valid-temperat: place
-		Start: 0
 	build-invalid: place
+		Start: 1
 	target-select: place
 	target-valid-desert: place
```

Comment:
This and the entry below it aren't wanted/needed.  Could you please remove them too?